Automattic/sensei
# Update jsx extensions to js

### Changes proposed in this Pull Request

* Renaming `jsx` extensions to `js` in Setup Wizard.
  * The goal of this change is to have the correct references to the source files in the [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/sensei-lms/dev/pt-br/default/).

### Testing instructions

* Run `npm run build` and make sure it works properly without errors.
* Run `npm run lint-js` and make sure it works properly without errors.
* Run `npm run test-js` and make sure it works properly without errors.
* Navigate through the Setup Wizard and make sure it works properly without errors.
